### PR TITLE
SDL2 Frontend: Use argv[0], add a --version, and reorder options

### PR DESCRIPTION
**Pull request by linkmauve**

### CODE REVIEW

File: src/citra/citra.cpp [hunk 1]
```diff
@@ -34,11 +35,12 @@
 #include "video_core/video_core.h"
 
 
-static void PrintHelp()
+static void PrintHelp(const char *argv0)
 {
-    std::cout << "Usage: citra [options] <filename>" << std::endl;
-    std::cout << "--help, -h            Display this information" << std::endl;
-    std::cout << "--gdbport, -g number  Enable gdb stub on port number" << std::endl;
+    std::cout << "Usage: " << argv0 << " [options] <filename>" << std::endl;
```
Comment (yuriks) [inline src/citra/citra.cpp]: Can you make help print the version too? I don't think there's any reason not to.

File: src/citra/citra.cpp [hunk 2]
```diff
@@ -34,11 +35,12 @@
...
+    std::cout << "-g, --gdbport=NUMBER  Enable gdb stub on port NUMBER" << std::endl;
```
Comment (yuriks) [inline src/citra/citra.cpp]: You can make these just a single string (either raw string literal or by concatting several string literals)

**Comment by yuriks** [comment]
LGTM

File: src/citra/citra.cpp (hunk 1 above)
Comment (linkmauve) [inline src/citra/citra.cpp]: This isn’t something most programs do, from a relatively small panel of tested programs. This help is there to give the user the most commonly used arguments, --version on the other hand just gives the version and exits.

**Comment by bunnei** [comment]
LGTM

**Pull request merged**